runtime/src/mem/jemalloc/mem-jemalloc.c

```diff
@@ -47,11 +47,12 @@
 // the performance difference in a playground just to see if it's worth
 // investigating further.
 
-// TODO put these into a struct?
-static void*  heap_base = NULL;
-static size_t heap_size = 0;
-static size_t cur_heap_offset = 0;
-static pthread_mutex_t chunk_alloc_lock;
+static struct shared_heap {
+  void* base;
+  size_t size;
+  size_t cur_offset;
+  pthread_mutex_t alloc_lock;
+} heap; // static, will be "zero" initialized automatically
```

Comment:
I originally did a partial initialization along the lines of  `} heap = {NULL, 0, 0 /*, default init mutex*/}`, but I didn't really like that. pthread_mutex_t is an opaque type so we can't actually specify it's initializer.  This way we just let the compiler pick the default initializer for each type, instead of specifying a partial list.